presentations/proposed/scipy2022/maintainers/proposal.md

```diff
@@ -48,3 +48,28 @@ Track: Maintainers
 Author: Pamphile T. Roy
 Keywords: Scientific Python, Community
 Type:  Talk
+corresponding author
+
+### Author 2
```

Comment:
Then I would do this as well. Also not sure what this char means `†`.

If you have personal websites, we can also add these.


```suggestion
corresponding author

### Author 1

First name†: Pamphile T.
Last name: Roy
Email: roy.pamphile@gmail.com 	
Country/region: Austria
Organization: Quansight

### Author 2
```